js/src/tooltip.js

```diff
@@ -465,6 +463,10 @@ class Tooltip extends BaseComponent {
     return offset
   }
 
+  _parseMaybeFunction(content) {
+    return typeof content === 'function' ? content.call(this._element) : content
+  }
+
```

Comment:
Not sure but there may be a better name here :slightly_smiling_face:

Reply:
_invokeOrDefaultTo, _callOrFallbackValue, _runFuncPerhapsReturnPassedValue, _executeOrReturn, _getReturnOrUseDefault

lol :-D